packages/evolution-generator/src/common/validations.tsx

```diff
@@ -259,7 +259,9 @@ export const postalCodeValidation: Validations = (value) => {
             }
         },
         {
-            validation: !/^[A-Za-z][0-9][A-Za-z]( )?[0-9][A-Za-z][0-9]\s*$/.test(String(value)),
+            // To be valid in Canada, the postal code cannot have the letters D, F, I, O, Q, or U. It also cannot have W or Z in the first character.
+            // See: https://en.wikipedia.org/wiki/Postal_codes_in_Canada#Number_of_possible_postal_codes
+            validation: !/^[abceghj-nprstvxy][0-9][abceghj-nprstv-z]( )?[0-9][abceghj-nprstv-z][0-9]\s*$/i.test(String(value)),
```

Comment:
Technically, evolution is not a _canadian_ platform, so we should not have a country specific postal code validation here. But it was like that before. Maybe just add a FIXME here to say that instead of returning the validation, this can be a function, which takes the country as parameter and return the correct postal validation.